apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/remote/TLSChannelBuilder.java

```diff
@@ -37,10 +38,12 @@
     @Override
     public NettyChannelBuilder build(
         NettyChannelBuilder managedChannelBuilder) throws AgentPackageNotFoundException, SSLException {
-        File caFile = new File(AgentPackagePath.getPath(), CA_FILE_NAME);
-        if (caFile.exists() && caFile.isFile()) {
+        if (Config.Agent.FORCE_TLS) {
             SslContextBuilder builder = GrpcSslContexts.forClient();
-            builder.trustManager(caFile);
+            File caFile = new File(AgentPackagePath.getPath(), CA_FILE_NAME);
```

Comment:
I think if CA file exists, there is no need to change the `FORCE_TLS=true` manually.

Reply:
So either CA file exists or `FORCE_TLS=true`, TLS should be used?

Reply:
Yes, we don't have to set both 2 settings.